doc/nrf/libraries/modem/nrf_modem_lib.rst

```diff
@@ -102,6 +103,10 @@ Complete the following steps to configure the modem trace level at compile time:
 The application can use the :c:func:`nrf_modem_lib_trace_level_set` function to set the desired trace level.
 Passing ``NRF_MODEM_LIB_TRACE_LEVEL_OFF`` to the :c:func:`nrf_modem_lib_trace_level_set` function disables trace output.
 
+.. note::
+   The current trace level is stored in the modem on ``AT+CFUN=0``.
```

Comment:
I don't think it is correct to use the `AT+CFUN=0` command to store the trace level. It is more something that occur when CFUN=0 is called.  I'll rewrite this.

Reply:
What `AT+CFUN=0` does is to persist the current config in the modem, so it will use the same setting after a reboot. The modem will also persist the config on a fixed interval, but I don't remember how many hours between each time.